scss/_variables.scss

```diff
@@ -488,9 +488,11 @@ $border-color:                $gray-300 !default;
 // scss-docs-end border-variables
 
 // scss-docs-start border-radius-variables
-$border-radius:               .25rem !default;
-$border-radius-sm:            .2rem !default;
-$border-radius-lg:            .3rem !default;
+$border-radius:               .375rem !default;
```

Comment:
Aren't these changes breaking ones?

Reply:
I don't see any of these as breaking changes, no.

Reply:
To my eyes, it's definitely a breaking change in the core :/

Reply:
There will be other visual changes as well coming in v5.2.0, but none of them are breaking. Nothing functionally changes, and due to how our variables are re-used and extended, things should work relatively seamlessly.

Reply:
I agree it's not BC but it should be highlighted since it'll change appearance of most of our components, even slightly (thus will lead to opened issues).

Reply:
I am not intending to keep back this PR, and of course is handy to have more helpers.

The only thing I would think of, is to introduce the new variables and helpers but trying to keep the old values and a variation between them.

In this form, it will affect all these sites that have used the default bootstrap.css (and maybe are many)

Reply:
Let's say someone is using the `$border-radius` variable in a calculation, e.g. `$border-radius * 10`. The change is 50% of what it was, and it will be noticeable.